docs/index.html

```diff
@@ -0,0 +1,2007 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ecosystem website</title>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/tile.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/button.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/list.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/tag.min.js"></script>
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/themes.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/grid.css" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="cds-theme-zone-white custom-theme">
+
+      <header>
+        <div class="inside-header cds--grid layout-lead-space-fixed__container">
+          <img class="ecosystem-logo" src = "ecosystem-logo.svg" width="350px" alt="Qiskit Ecosystem"/>
```

Comment:
I recommend running Prettier on this file manually. You can do `npx prettier --write docs/index.html`. The `src = ` shouldn't have space around it like this.

Reply:
Nice! I have run the Prettier on each template instead of the `index.html`. The only "problem" is with the tabs, so maybe running the Prettier on the `index.html` after building the website is not a bad idea either.

Reply:
I didn't realize this was generated when I left the comment. I do not recommend running Prettier manually on a generated file. That would require that anyone in the future who generates the file needs to also run Prettier to avoid a huge diff in Git.

That would be fine if we required using Prettier via CI. But we shouldn't do that. It's too much complexity for too little gain.

So ignore me and go back to not using Prettier here.